examples/cloud-router-2-aws-connection/main.tf

```diff
@@ -1,8 +1,13 @@
 provider "equinix" {
   client_id     = var.equinix_client_id
   client_secret = var.equinix_client_secret
+  endpoint      = var.equinix_endpoint
```

Comment:
Endpoint shouldn't be included for the customer. They will almost always use the default endpoint.